modules/system/assets/ui/less/toolbar.less

```diff
@@ -27,6 +27,23 @@
     &:after {
         right: -8px;
     }
+	
+	    .clear-input-text {
+				font-size:large;
+				color:#666666;
+				display: inline-block;
+				position: absolute;
+				right: 1px;
+				top: 2px;
+				background-color: #fff;
+				padding-left:6px;
+				height:30px;
+				border: 0;
+			}
+			
+		.clear-input-text:focus {
+			outline:none;
+		}
```

Comment:
Please fix the indentation on these styles, should be four spaces and inline with the others in the same nested context